src/traits/window.rs

```diff
@@ -6,8 +6,21 @@ use glib::translate::{from_glib_none, ToGlibPtr};
 use ffi;
 use glib::to_gboolean;
 use cast::GTK_WINDOW;
+use gdk;
 
 pub trait WindowTrait : ::WidgetTrait {
+    fn move_(&self, x: i32, y: i32) {
```

Comment:
Generally `int`s might be architecture dependent and that must be why there exists `c_int` (so `isize` would be a mistake).